state_test.go

```diff
@@ -310,35 +310,26 @@ func TestState_GetSequence(t *testing.T) {
 
 func TestState_getCommittedSeals(t *testing.T) {
 	pool := newTesterAccountPool()
-	pool.add("A", "B", "C", "D")
+	pool.add("A", "B", "C", "D", "E")
 
 	s := newState()
 	s.validators = pool.validatorSet()
 
 	s.addCommitted(createMessage("A", MessageReq_Commit))
 	s.addCommitted(createMessage("B", MessageReq_Commit))
+	s.addCommitted(createMessage("C", MessageReq_Commit))
 	committedSeals := s.getCommittedSeals()
 
-	assert.Len(t, committedSeals, 2)
-	assert.True(t, &committedSeals[0] != &s.committed["A"].Seal)
-	foundSeal := false
-	for _, seal := range committedSeals {
-		if reflect.DeepEqual(seal, s.committed["A"].Seal) {
-			foundSeal = true
-		}
+	assert.Len(t, committedSeals, 3)
+	processed := map[NodeID]bool{}
```

Comment:
The better way is to use map[NodeID]struct{}

Reply:
i think it is easier this way otherwise we need to use this code in actual test
```
_, exists := processed[commSeal.NodeID]
assert.False(t, exists) 
```
But if you think it is necessary we will change it

Reply:
No, just put my thougts.

Reply:
I agree with @b00ris. `bool` does not represent anything there and unless semantically necessary, I prefer to use `struct{}`

Reply:
changed